src/bin/lib/plan-messages.ts

```diff
@@ -22,24 +23,37 @@ const renderPlan = (batches: RequestBatch[]) => {
   for (const batch of batches) {
     renderBatch(batch)
 
-    if (batch.errors.length > 0 || batch.contentTransformErrors.length > 0) {
+    if (batch.validationErrors.length > 0) {
       console.log('\n')
     }
 
-    for (const error of batch.errors) {
+    for (const error of batch.validationErrors) {
       console.log(chalk`{red.bold Error: ${error.message}}`)
     }
 
-    for (const transformError of batch.contentTransformErrors) {
-      console.log(transformError)
-      console.log('\n')
-    }
-
     if (batch.intent.shouldPublish()) {
       console.log(chalk`\n{bold.underline Publish Content Type} {bold.yellow ${batch.intent.getContentTypeId()}}`)
     }
+  }
+}
 
+const renderValidationErrors = (batches: RequestBatch[]) => {
+  console.log(chalk`{bold.red The following migration has been planned but cannot be run because it contains errors}\n\n`)
+  renderPlan(batches)
+}
+
+const renderRuntimeErrorsAndWriteToFile = async (batches: RequestBatch[], filename): Promise<void> => {
```

Comment:
Related to some other comment I made, wdyt about extracting the `writeErrorsToLog(batch.runtimeErrors, filename)` logic to somewhere else? So the function does just one thing.